examples/column_transformer_mixed_types.py

```diff
@@ -0,0 +1,104 @@
+"""
+===================================
+Column Transformer with Mixed Types
+===================================
+
+This example illustrates how to apply different preprocessing and
+feature extraction pipelines to different subsets of features,
+using :class:`sklearn.compose.ColumnTransformer`.
+This is particularly handy for the case of datasets that contain
+heterogeneous data types, since we may want to scale the
+numeric features and one-hot encode the categorical ones.
+
+In this example, the numeric data is standard-scaled after
+mean-imputation, while the categorical data is one-hot
+endcoded after imputing missing values with a new category
+(:code:`'missing'`).
+
+Finally, the preprocessing pipeline is integrated in a
+full prediction pipeline using :class:`sklearn.pipeline.Pipeline`,
+together with a simple classification model.
+"""
+
+# Author: Pedro Morales <part.morales@gmail.com>
+#
+# License: BSD 3 clause
+
+from __future__ import print_function
+
+import pandas as pd
+
+from sklearn.compose import make_column_transformer
+from sklearn.pipeline import make_pipeline
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import StandardScaler, CategoricalEncoder
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split, GridSearchCV
+
+
+# Read data from Titanic dataset.
+titanic_url = ('https://raw.githubusercontent.com/amueller/'
+               'scipy-2017-sklearn/master/notebooks/datasets/titanic3.csv')
+data = pd.read_csv(titanic_url)
+
+# We will train our classifier with the following features:
+# Numeric Features:
+# - age: float.
+# - fare: float.
+# Categorical Features:
+# - embarked: categories encoded as strings {'C', 'S', 'Q'}.
+# - sex: categories encoded as strings {'female', 'male'}.
+# - plcass: categories encoded as ints {1, 2, 3}.
+num_feats = ['age', 'fare']
+cat_feats = ['embarked', 'sex', 'pclass']
```

Comment:
Better be explicit and call those variables: `numerical_features` and `categorical_features`. Readability is extra important for examples.

Reply:
I agree.